src/PantherTestCaseTrait.php

```diff
@@ -129,14 +128,19 @@ public static function isWebServerStarted(): bool
 
     public function takeScreenshotIfTestFailed(): void
     {
-        if (!\in_array($this->getStatus(), [BaseTestRunner::STATUS_ERROR, BaseTestRunner::STATUS_FAILURE], true)) {
+        // provided by PHPUnit\Framework\TestCase
+        if (!method_exists($this, 'status')) {
```

Comment:
Could you adapt this to support both PHPUnit 10 and older versions?

Reply:
done: https://github.com/arderyp/panther/commit/e7b35205822b079227382791109779772cc8743b

@dunglas resolve if this is sufficient